library/include/internal/hipsolver-types.h

```diff
@@ -126,6 +125,12 @@ typedef enum
     HIPSOLVER_EIG_RANGE_I   = 223,
 } hipsolverEigRange_t;
 
+typedef enum
+{
+    HIPSOLVER_DETERMINISTIC_RESULTS           = 241,
+    HIPSOLVER_ALLOW_NON_DETERMINISTIC_RESULTS = 242,
+} hipsolverDeterministicMode_t;
```

Comment:
There have been requests to ensure compatibility between hipSOLVER and cuSOLVER all the way down to the values of enumerations. We should consider whether we want to adopt matching values immediately for new functionality, or keep following our existing pattern until there's a good time to change them all at once.

Reply:
Is that something we should wait for a major release to change, or can it be changed anytime?

Reply:
Changing it would be an ABI change (though not an API change). It would have to wait until at least ROCm 7.0.